### DISCUSSION TITLE
Comic Sans?

### DISCUSSION THREAD

**Discussion by Madis0** [C1710/blobmoji#64] [Style]
What's up with the use of Comic Sans in all text-including emoji? Why not use something more Google-y like Product Sans or Roboto?

**Comment by C1710** [comment]
So first of all, this is [Comic Neue](comicneue.com) which is a more readable and appealing version of Comic Sans.  
The second thing is that this fork is not affiliated with Google, so you will probably not see new content with connections to Google.  
I mainly chose Comic Neue because I think it's a more cartoonish style than fonts like Roboto which I think is better suited for an emoji font.  
Just don't get me wrong here - I _love_ Roboto but in my opinion, something like Comic Neue is just a better choice in this case.  
However, fell free to try other fonts!
In most cases I haven't converted the text fields to paths, so you can change the font very easily.  
Ans since an SVG file is basically an plaintext XML file, you could just try to write a little script replacing all occurences of the word `Comic Neue` with `Roboto` or something else.  
But please be aware that some text fields _are_ converted to paths. In this case you'll have to delete these paths and replace them with actual text fields again.

**Comment by C1710** [comment]
But I like it how people start questioning my decisions 😅

**Comment by Madis0** [comment]
>The second thing is that this fork is not affiliated with Google, so you will probably not see new content with connections to Google.

True, but it seems like you do want to keep the style faithful, yet consistent with others.

>I mainly chose Comic Neue because I think it's a more cartoonish style than fonts like Roboto which I think is better suited for an emoji font.
Just don't get me wrong here - I love Roboto but in my opinion, something like Comic Neue is just a better choice in this case.

Quoting you:
>My goal is to upgrade the Blob emojis with a fresh style which is consistent to other emoji vendors.

Here's one example of a text-based emoji - ["back arrow"](https://emojipedia.org/back-with-leftwards-arrow-above/). By the images, only Messenger has a cartoonish font, however even that [has been discontinued](https://blog.emojipedia.org/facebook-discontinues-messenger-emojis/).

So I would suggest considering using a simpler, more generic font in your emoji style too :slightly_smiling_face: 

>But I like it how people start questioning my decisions :sweat_smile:

Well, it has been turned into a [Magisk module](https://github.com/Magisk-Modules-Repo/magisk-blobmoji) after all, so your decidions affect many blob fans :grinning:

**Comment by C1710** [comment]
Well, this consistency I mean is rather about the actual _meaning_ of the emojis.  
Take [this](https://emojipedia.org/face-with-tears-of-joy/) as a very bad example (there are better examples but I couldn't find a better one yet).  
In the older version this Blob was grinning, while it has an open mouth on (almost) all platforms.  
A different font (as long as it's not something liek Webdings) won't change anything about the meaning of an emoji.  
But I'll think about a solution :smile:  

> Well, it has been turned into a Magisk module after all, so your decidions affect many blob fans 😀  

Wow, I didn't know about that one. 😂 But I don't use any device with Magisk and on my tablet (the only Android device on which I can actually exchange the emoji font) I can simply "install" a new version using TWRP 😁

**Comment by C1710** [comment]
So here's an overview of the SVG files which need to be modified manually (at least currently. I think I'll make it easier in the future).  
If you got GNU/Linux on your PC (or Windows Subsystem for Linux), you can simply `cd` into the SVG directory and run this command: `sed -i -- 's/Comic Neue/Roboto/g' *.svg`. It will replace all occurences of `Comic Neue` in the SVG files with `Roboto`. You'll only need to render the PNG files.  
I'll write an introduction soon which makes rendering the PNG files relatively easy.
[problems.txt](https://github.com/C1710/blobmoji/files/1943968/problems.txt)

**Comment by C1710** [comment]
Okay. The only emojis you'll need to fix manually are the five book emojis.

**Comment by Madis0** [comment]
So I've tested the command and other than some alignment problems (the boxy symbols like blood type need a bit smaller font) it looks sharp. I'll have to admit, I liked `U+2139` :information_source:  in Comic Neue though.

Either way, I found some more emoji that could use font consistency:

* `U+1F524` `U+1F523` `U+1F522` `U+1F521` `U+1F520` (input types)
* `U+1F3EA` `U+1F3E9` `U+1F3E8` `U+1F3E7` `U+1F3E6` (buildings)
* and every regional letter for making flags
